src/include/access/k2/pg_schema.h

```diff
@@ -37,6 +37,8 @@ Copyright(c) 2022 Futurewei Cloud
 namespace k2pg {
     typedef int32_t ColumnId;
     constexpr ColumnId kFirstColumnId = 0;
+    // Additional partition keys TableId and ColumnId added to each table
```

Comment:
should be TableId and IndexId, not TableId and ColumnId

Reply:
Removed the change.